### PR TITLE
Python 3.9

### PR DESCRIPTION
**Pull request by xyb**

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -54,6 +54,12 @@ environment:
     - PYTHON: "C:\\Python38-x64"
       PYTHON_VERSION: "3.8"
       PYTHON_ARCH: "64"
+    - PYTHON: "C:\\Python38"
+      PYTHON_VERSION: "3.9"
+      PYTHON_ARCH: "32"
+    - PYTHON: "C:\\Python38-x64"
+      PYTHON_VERSION: "3.9"
+      PYTHON_ARCH: "64"
```
Comment (ifduyue) [inline .appveyor.yml:62]: It seems that appveyor have installed Python 3.9 only for their Visual Studio 2019 image. If we want to test with Python 3.9 we'll have to use Visual Studio 2019 image.

Comment (xyb) [reply .appveyor.yml:63]: Yes, I have updated the configuration.

**Comment by ifduyue** [comment]
@xyb merged, thanks!

**Comment by xyb** [comment]
@ifduyue sorry, just notice that Travis did not create bin wheels of python 3.9 for Linux and Mac, but I do not know how to fix it.

**Comment by xyb** [comment]
Don't know why Travis installed cibuildwheel==1.1.0, but cibuildwheel supports python 3.9 until version 1.6.2. I may update another PR to fix this.

**Comment by xyb** [comment]
I have pushed #41 to update cibuildwheel hope to fix this.

**Pull request merged**